packages/server/src/main/java/software/uncharted/terarium/hmiserver/utils/rebac/askem/RebacUser.java

```diff
@@ -9,44 +9,45 @@
 import software.uncharted.terarium.hmiserver.utils.rebac.SchemaObject;
 
 public class RebacUser extends RebacObject {
-	private ReBACService reBACService;
+	private final ReBACService reBACService;
 
-	private boolean serviceUser;
+	private final boolean serviceUser;
 
-	public RebacUser(String id, ReBACService reBACService) {
+	public RebacUser(final String id, final ReBACService reBACService) {
 		super(id);
 		this.reBACService = reBACService;
 
-		serviceUser = reBACService.isServiceUser(id);
+		serviceUser = ReBACService.isServiceUser(id);
```

Comment:
Why did this case change? The local variable is "reBACService"

Reply:
This looks like the call is being done statically on ReBACService the class... is that right?

Reply:
yeah